public/app/core/components/EmptyListCTA/EmptyListCTA.test.tsx

```diff
@@ -7,6 +7,7 @@ const model = {
   buttonIcon: 'ga css class',
   buttonLink: 'http://url/to/destination',
   buttonTitle: 'Click me',
+  onClick: 'handler',
```

Comment:
What's the reason for p[assing a string into the onClick prop? Wouldn't it make more sense to user `jest.fn()`?

Reply:
Good point, will update it.